src/app/components/form/input/FormInputEmailAddress.vue

```diff
@@ -23,14 +25,14 @@
       </FormInputStateError>
     </template>
     <template #stateWarning>
-      <FormInputStateWarning
```

Comment:
Let's keep this a warning here, email addresses can be formatted in wild ways, see https://www.dominicsayers.com/isemail/
The current implementation intentionally keeps the warning open with "Make sure this is correct", not saying the email is incorrect because we decide so, but to make sure users don't submit typos.